src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -29,6 +32,11 @@ public sealed class Schema : ISchema
         /// </summary>
         public int ColumnCount => _columns.Length;
```

Comment:
>ColumnCount [](start = 19, length = 11)

I see you're in the process of replacing `ColumnCount` with `Count` throughout the codebase. Is that so? Were you intending to finish that work in a future PR? (Totally fine if so, just wanted to know your plans.)

Reply:
Yes, future PR. I got 200+ errors when I made `ColumnCount` an explicit `ISchema` method, so I reverted for now

---
In reply to: [235110357](https://github.com/dotnet/machinelearning/pull/1680#discussion_r235110357) [](ancestors = 235110357)